test/mysql_tests.erl

```diff
@@ -1005,3 +1000,14 @@ parse_db_version(Version) ->
   [Version1 | _] = binary:split(Version, <<"-">>),
   lists:map(fun binary_to_integer/1,
             binary:split(Version1, <<".">>, [global])).
+
+is_access_denied({1045, <<"28000">>, <<"Access denie", _/binary>>}) ->
+    true; % MySQL 5.x, etc.
+is_access_denied({1698, <<"28000">>, <<"Access denie", _/binary>>}) ->
+    true; % MariaDB 10.3.15
+is_access_denied({1251, <<"08004">>, <<"Client does not support authentication "
+                                       "protocol requested by server; consider "
+                                       "upgrading MariaDB client">>}) ->
```

Comment:
Without re-checking (still on my mobile instead of a real computer), the same error message (with "MySQL" instead of "MariaDB") occurs if MySQL 8 is used and the default auth plugin is set to `mysql_native_password` (vs `caching_sha2_password`) on the server. While this is probably of no consequence for current tests, it may be worth to truncate the expected message before "MariaDB", or at least to keep it in mind for the future.

Reply:
Good point! I think it's good to make sure the tests pass with this configuration even though we don't run it with travis.

Reply:
Good point! I think it's good to make sure the tests pass with this configuration even though we don't run it with travis.